NEventstore.Persistence.DocumentDB/Constraints/Logical_Key.js

```diff
@@ -0,0 +1,57 @@
+﻿function LogicalKey_Constraint() {
+    var context = getContext();
+    var collection = context.getCollection();
+    var request = context.getRequest();
+    var docToUpdate = request.getBody();
+
+    HasDuplicates();
+
+    function HasDuplicates(continuation)
```

Comment:
is the continuation stuff needed as we have no need to page (we'll error as soon as the first result is found)